docs/creating_component_index.md

```diff
@@ -3,17 +3,34 @@ title: "Creating components"
 sidebar_label: "Introduction"
```

Comment:
I think this needs a better name than Introduction, as it's not the first page in the section.

Reply:
Yeah, that's weird. Fixed.